model/job.go

```diff
@@ -225,6 +234,123 @@ func (j *Job) loadJobSpec() (err error) {
 	return nil
 }
 
+// MergeSpec is used to merge temporary spec patches into each job. otherJob should only be
+// the hcf/patch-properties job.
+func (j *Job) MergeSpec(otherJob *Job) {
+	// Ignore otherJob.Name, otherJob.Description
+	if otherJob.Packages != nil {
+		j.Packages = append(j.Packages, otherJob.Packages...)
```

Comment:
The fact that packages and properties are shared across jobs (the new ones, at least) is not an issue, right?

What happens if some of the new packages/properties already exist in the job being modified? (This needs a comment in code to describe why it's okay)